python/pyspark/broadcast.py

```diff
@@ -82,7 +82,7 @@ def dump(self, value, f):
         except pickle.PickleError:
             raise
         except Exception as e:
-            msg = "Could not serialize broadcast: " + e.__class__.__name__ + ": " + e.message
+            msg = "Could not serialize broadcast: " + e.__class__.__name__ + ": " + str(e)
```

Comment:
BTW, we should keep in mind that

```python
>>> "" + Exception(u"jörn").message
u'j\xf6rn'
>>> "" + str(Exception(u"jörn"))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
UnicodeEncodeError: 'ascii' codec can't encode character u'\xf6' in position 1: ordinal not in range(128)
```

this could cause such encoding problem. It'd be really rare case though I think. It seems this usage is already there in the code base so I assume it is fine.